doc/main.doc.md

```diff
@@ -64,27 +64,53 @@ Each module `xxx` has a `xxx.init.s` file with its initialization code.
 
 ## Main Loop
 
-Do nothing, we work in `VBlank`.
+The main logic in included in `body.s`.
+
+We use a look to synchronize the update of the values with the update of the screen. After each iteration we wait for a VBlank, and if we didn't finished the iteration, the VBlank interruption do nothing.
 ~~~
 ; ///////// MAIN LOOP \\\\\\\\\
 loop:
-	jr loop
+; //// WAIT FOR VBLANK \\\\
+	halt
+	ld a,(lock)

```

Comment:
`lock` is a bit vague for a global variable name

Reply:
Actually it's not global, it's local to the main file. (no global_ before) but yes, does VBlqnk_lock sound good?